framework/src/st_things/things_stack/easy-setup/easysetup_manager.c

```diff
@@ -39,6 +39,8 @@
 #include "utils/things_thread.h"
 #include "utils/things_rtos_util.h"
 
+#include <time.h>
```

Comment:
Please move this at location of standard header including. (line 29)

Reply:
OK Thanks.
I will fix it